libtransmission/peer-mgr.cc

```diff
@@ -76,12 +76,13 @@ class HandshakeMediator final : public tr_handshake::Mediator
             return {};
         }
 
-        auto info = TorrentInfo{};
-        info.info_hash = tor->info_hash();
-        info.client_peer_id = tor->peer_id();
-        info.id = tor->id();
-        info.is_done = tor->is_done();
-        return info;
+        return TorrentInfo{
+            tor->info_hash(), // info_hash
```

Comment:
I wonder if we could bump to C++20, not requiring full compliance but at least allowing for use of designated initializers… Different (and non-blocking) topic though.

Reply:
I support that, In fact, I raised #5854 some time ago to track this.